build_java.gy

```diff
@@ -413,7 +414,9 @@ mkdir -p "${ARTIFACTS_DIR}"
                         stage('OSX: Test') {
                             steps {
                                 // Clean the workspace
-                                cleanWs(deleteDirs: true)
+                                cleanWs(deleteDirs: true,
+                                disableDeferredWipeout: true,
+                    			patterns:[[pattern: '*.log', type: 'EXCLUDE']]))
```

Comment:
What is the behavior of this?

Reply:
Will it delete everything except *.log?

Reply:
It'll clean the workspace on Jenkins, except for file with log extension. disableDeferredWipeout is recommended by Ming to make sure the pipeline takes the time to delete the workspace, it should affect the build a lot.